prow/pluginhelp/externalplugins/externalplugins.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package externalplugins provides the plugin help components to be compiled into external plugin binaries.
+// Since external plugins only need to serve a "/help" endpoint this package just provides an
+// http.HandlerFunc that wraps an ExternalPluginHelpProvider function.
+package externalplugins
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"k8s.io/test-infra/prow/pluginhelp"
```

Comment:
nit: we usually have imports in order (stdlib, third-party, self)

Reply:
fixed.